app/src/main/res/values/settings_keys.xml

```diff
@@ -191,6 +189,38 @@
         <item>@string/audio_webm_key</item>
     </string-array>
 
+    <string name="left_gesture_control_key">left_gesture_control</string>
+    <string name="default_left_gesture_control_value">@string/left_brightness_control_key</string>
+    <string name="left_brightness_control_key">left_brightness_control</string>
+    <string name="left_volume_control_key">left_volume_control</string>
+    <string name="left_none_control_key">left_none_control</string>
+    <string-array name="left_gesture_control_description">
+        <item>@string/brightness</item>
+        <item>@string/volume</item>
+        <item>@string/none</item>
+    </string-array>
+    <string-array name="left_gesture_control_values">
+        <item>@string/left_brightness_control_key</item>
+        <item>@string/left_volume_control_key</item>
+        <item>@string/left_none_control_key</item>
+    </string-array>
+
+    <string name="right_gesture_control_key">right_gesture_control</string>
+    <string name="default_right_gesture_control_value">@string/right_volume_control_key</string>
+    <string name="right_brightness_control_key">right_brightness_control</string>
+    <string name="right_volume_control_key">right_volume_control</string>
+    <string name="right_none_control_key">right_none_control</string>
```

Comment:
Why did you duplicate these? I'd just call them "brightness_control_key", "volume_control_key" and "none_control_key" and use them both for right and left.